packages/connectkit/src/components/Common/ChainSelectList/index.tsx

```diff
@@ -68,146 +68,150 @@ const ChainSelectList: React.FC = () => {
         <ChainButtons>
           {chains.map((x) => {
             const c = supportedChains.find((ch) => ch.id === x.id);
-            const ch = { ...c, ...x };
-            return (
-              <ChainButton
-                key={`${ch?.id}-${ch?.name}`}
-                disabled={
-                  !switchNetwork ||
-                  ch.id === chain?.id ||
-                  (isLoading && pendingChainId === ch.id)
-                }
-                onClick={() => switchNetwork?.(ch.id)}
-                style={{
-                  opacity:
-                    !switchNetwork && ch.id !== chain?.id ? 0.4 : undefined,
-                }}
-              >
-                <span
+
+            if (c) {
```

Comment:
This new if statement makes this a bit hard to review within this diff.
Might be better if this was just `if(!c) return`

Reply:
Got it!